plat/intel/soc/stratix10/platform_def.h

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __PLATFORM_DEF_H__
+#define __PLATFORM_DEF_H__
+
+#include <arch.h>
+#include <gic_common.h>
+#include <interrupt_props.h>
+#include <tbbr/tbbr_img_def.h>
+#include  <bl_common.h>
+#include <common_def.h>
+
+
+/*******************************************************************************
+ * User configuration
+ ******************************************************************************/
+#define ENABLE_HANDOFF
```

Comment:
What is ``ENABLE_HANDOFF`` meant to do? I don't see it anywhere. If it does nothing, it should be removed.